app/src/main/java/org/oppia/app/player/state/itemviewmodel/DragAndDropSortInteractionViewModel.kt

```diff
@@ -18,25 +20,33 @@ class DragAndDropSortInteractionViewModel(
   interaction: Interaction,
   private val interactionAnswerErrorOrAvailabilityCheckReceiver: InteractionAnswerErrorOrAvailabilityCheckReceiver
 ) : StateItemViewModel(ViewType.DRAG_DROP_SORT_INTERACTION), InteractionAnswerHandler,
-  OnItemDragListener {
+  OnItemDragListener, OnDragEndedListener {
```

Comment:
ktlint failing for this file, check using similar command as above with this file path.

Reply:
done